tensorflow/core/ops/array_ops.cc

```diff
@@ -2292,6 +2292,24 @@ Status SliceHelper(InferenceContext* c, ShapeHandle begin_value,
   return Status::OK();
 }
 
+Status SliceHelper(InferenceContext* c, ShapeHandle begin_value,
```

Comment:
can you comment this function?  Maybe a different name than SliceHelper too would help make it more clear what it does.

Reply:
Thanks @cwhipkey. Done.